examples/jsm/renderers/CSS2DRenderer.js

```diff
@@ -18,6 +18,17 @@ class CSS2DObject extends Object3D {
 		this.element.style.userSelect = 'none';
 
 		this.element.setAttribute( 'draggable', false );
+		
+		const _this = this;
+		this.addEventListener( 'added', function() {
```

Comment:
I'm not sure this is the right fix for the problem. 

I wonder if it wouldn't be better to solve this issue on `Object3D` level. Meaning when a 3D object is added or removed to/from the scene graph, descendants receive an event as well since they are indirectly affected by this operation.

Reply:
I believe having this particular event logic in the `CSS2DObject` level provides a unique solution for objects that also contain HTML DOM references.

`CSS2DObject` types do not have descendants, so it makes sense to require event listeners to their parent rather than assigning events externally.

Reply:
> CSS2DObject types do not have descendants

I don't think this assumptions is true.

Reply:
You are right, my assumption was not true. I tried nesting a `CSS2DObject` within another `CSS2DObject` and they render HTML on top of each other (not pretty, but still optional). The solution I provided in this PR still work when ancestors invoke `clear()` or `removeFromParent()`.

Do you have any pseudo code in mind for adding events to descendants within the `Object3D` class?

Reply:
> Meaning when a 3D object is added or removed to/from the scene graph, descendants receive an event as well since they are indirectly affected by this operation.

Related issue: #22416
Related PR: #16934